docs/features.rst

```diff
@@ -16,10 +16,6 @@ Admin interface screenshots
 
    Managing email domains
 
-.. figure:: assets/screenshots/status.png
```

Comment:
Can you also remove the mentioned image? I don't think we need it anymore after deleting this reference.

Reply:
Sure thing.  I have done that and it is now showing in this PR.

Reply:
thx :)